inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/FilterFunctionUtils.java

```diff
@@ -91,6 +92,32 @@ public static List<FilterFunction> createFilterFunctions(FilterDefinition filter
         return filterFunctions;
     }
 
+    public static FilterStrategy parseFilterStrategy(TransformResponse transformResponse) {
```

Comment:
Please add java doc for static method, thanks.

Reply:
done